WRITING TESTS.md

```diff
@@ -0,0 +1,141 @@
+# Writing and running tests on Deriv App
+
+This document is intended to be a guideline for writing and running tests on deriv app.
+
+Things that this document covers:
+
+* Brief explanation on test framework implemented on the project.(Jest)
+
+* We will describe how `react-testing-library` configured. 
+
+* An example of how to write test on using implemented framework
+
+   
+
+  ## Test Framework explanation 
+
+  ​	We are using `Jest` as test runner and [React Testing library](https://testing-library.com/docs/react-testing-library/intro/) as our main test component to have the ultimate needed utilities.
+
+  
+
+  ### 	Jest 
+
+  ​		With Jest we can run our tests quickly and even parallel.
+
+  ​		All of needed configurations are located in root folder of the app.
+
+  
+
+  #### 	Run Tests
+
+  ​		To Run tests from root directory simply run:
+
+  ​			```npm run test:jest```
+
+  ​		Be informed that this command will run all of the tests in the whole app based on following configuration provided.
+
+  ​		Also the mentioned command will generate a coverage directory in the root directory, you can check the current coverage percentage by the generated html.
```

Comment:
```suggestion
  ​		Also it will create a coverage directory in the project root so you can check the current coverage percentage from the generated html.
```